build.sbt

```diff
@@ -66,6 +66,8 @@ val kafkaScale = settingKey[Int]("Number of kafka docker containers")
 
 resolvers in ThisBuild ++= Seq(Resolver.bintrayRepo("manub", "maven"))
 
+// TODO read version or master
+val githubTree = "master"
```

Comment:
As Paradox does try to figure out the correct github tree itself, maybe we could reuse that value here as well? Or do you think it might be too hacky to rely on paradox property here?

Reply:
Yes, that seems hacky to me. I looked into copying the logic from there, but it was not completely straightforward.
How easy would it be to access the Paradox property?

Reply:
paradoxProperties.value.get(GitHubResolver.baseUrl)

Reply:
It gets more involved as that is available only in the `docs` project (which inherits settings from `CommonSettings`...
I went with duplicating the URL and some logic.